stable/gcloud-sqlproxy/templates/deployment.yaml

```diff
@@ -39,6 +39,9 @@ spec:
         {{ if $hasCredentials -}}
         - -credential_file=/secrets/cloudsql/{{ include "gcloud-sqlproxy.secretKey" . }}
         {{ end -}}
+        {{ if .Values.cloudsql.private -}}
+        - -ip_address_types=PRIVATE
+        {{ end -}}
```

Comment:
Can we instead just use `extraArgs` value and set it with `ip_address_types: PRIVATE` ?